ComposeStarter/app/src/main/java/com/example/android/wearable/composestarter/presentation/MainActivity.kt

```diff
@@ -187,6 +206,48 @@ fun Greeting(greetingName: String) {
     }
 }
 
+@Composable
+fun SampleDialog(
+    showDialog: Boolean,
+    onDismiss: () -> Unit,
+    onCancel: () -> Unit,
+    onOk: () -> Unit
+) {
+    val state = rememberResponsiveColumnState()
+
+    Dialog(
```

Comment:
What's the reasoning behind using standard `Dialog` with Horologist `AlertContent`, instead of Horologist `AlertDialog`?

Reply:
Screenshot tests. I think AlertDialog needs some tweaks somewhere to support screenshotting a dialog.  I'll follow up.